VAMobile/src/store/api/demo/mocks/personalInformation.json

```
@@ -0,0 +1,16 @@
+{
+    "/v2/user": {
+      "data": {
+        "id": "ae19ab8a-7165-57d1-a8e2-200f5031f66c",
+        "type": "user",
+        "attributes": {
+          "firstName": "KIMBERLY",
+          "middleName": "",
```

```suggestion
          "middleName": null,
```
With the new endpoint, if `middleName` isn't set, it would be `null` instead of an empty string